### ISSUE TITLE
FocusZone: Screen readers do not follow bidirectional keyboard navigation in scan mode

### ISSUE THREAD

**Issue by tonianelope** [microsoft/fluentui#15508]
### Environment Information

- **Package version(s)**: (^7.68.0)
- **Browser and OS versions**: (Microsoft Edge 86, Chrome 86, Firefox 81 on Windows 10 Enterprise)
- **ScreenReader versions**: (JAWS 2020, NVDA 2020.3, Narrator see OS)

### Describe the issue:

The bidirectional keyboard navigation does not work when using screen readers. *(maintainer edit: this issue only occurs in scan mode)*

#### Actual behavior:

Up/Down arrow keys will navigate through the items in dom order. Right/Left arrow keys goes through the letters of the item.

#### Expected behavior:

Items can be navigated with the arrow keys based on the visual layout as defined for the focus zone.

### If applicable, please provide a codepen repro:

The FluentUI list example is broken for all screen readers: 
https://developer.microsoft.com/en-us/fluentui#/controls/web/list
The FluentUI focusZone example is only working for JAWS:
https://developer.microsoft.com/en-us/fluentui#/controls/web/focuszone
### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? (No)

Requested priority: (Blocking)

Products/sites affected: (if applicable)

**Comment by paulgildea** [comment]
@tonianelope Thanks for filing this issue!

@smhigley FYI

**Comment by tomi-msft** [comment]
Hi @tonianelope! I've looked at this a bit more, and want to get a bit more understanding of the issue. I repro'd the issue using Narrator, and it seems that scan mode is enabled by default, which allows the Narrator to interact with non-tabable items. If you turn scan mode off, then you are able to use the keyboard to navigate through the different list items. This seems in line with the expected behavior that you are looking for. 

Is there a particular behavior that you are expecting when interacting with the screen reader?

**Comment by tonianelope** [comment]
@tomi-msft thanks for looking at this. 

My expectation was that if I defined a FocusZone for arrow key navigation, the screen reader would switch to the appropriate mode "automatically" to support it.
I'm putting "automatically" in quotes because I'm not familiar enough to know if it is possible for fluent-ui to force screen readers into the focus/form/input/non-scan mode. At least JAWS seems to do it under some circumstances, so that one at least seems to be possible to influence.

**Comment by tonianelope** [comment]
@tomi-msft any update on this?
